src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -322,6 +323,13 @@ private static SyntaxList<UsingDirectiveSyntax> GetUsings(SyntaxNode node)
             // some * "(" cases
             if (currentToken.Kind() == SyntaxKind.OpenParenToken)
             {
+                if (previousToken.Kind() == SyntaxKind.NewKeyword &&
+                    CSharpSyntaxFacts.Instance.IsDeclaration(previousToken.Parent))
```

Comment:
i'd really just prefer that we dont' do any sort of declaration check.  i'm not seeing the purpose.  either it's a construct where we should have 0 spaces, or we should have 1 space.  i feel like this 'declaratin' check can be removed and we can determine if it's 1 or 0 depending on if the parent is exactly the implicit-obj or constraint case.

I'm happy to make this change in the PR if you'd like.